CONTRIBUTING.md

```diff
@@ -65,6 +65,10 @@ Other Tips
 
         $ git commit --amend
 
+  - Then force push your branch:
```

Comment:
The `--force` is only necessary if one has already pushed the commit with `[skip ci]` -- but in that case it is also too late to change it. Instead, I think here in the documentation it would make more sense to edit the sentence above, to read:
```
If you already made the commit without including this string, but have not yet pushed it up, you can still edit ...
```